cmd/glasskube/cmd/bootstrap.go

```diff
@@ -37,6 +39,17 @@ var bootstrapCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		cfg, _ := cliutils.RequireConfig(config.Kubeconfig)
 		client := bootstrap.NewBootstrapClient(cfg)
+		ctx := cmd.Context()
+
+		currentContext := clicontext.RawConfigFromContext(ctx).CurrentContext
+
+		if !bootstrapCmdOptions.yes {
+			confirmMessage := fmt.Sprintf("Glasskube will be installed in context [%s].\nContinue? ", currentContext)
+			if !cliutils.YesNoPrompt(confirmMessage, false) {
```

Comment:
I think it was meant to be default `true`, since the "Y" is upper case in the issue description. Also minor, but could you remove the brackets around the context name?

Reply:
Done please check now @christophenne